net-misc/dahdi-tools/dahdi-tools-3.1.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 autotools
```

Comment:
You should be able to sort these alphabetically without breaking sourcing & running the ebuild.